webapp/client/src/pages/FahrkostenpauschalePersonAPage.js

```diff
@@ -32,7 +32,7 @@ FahrkostenpauschalePersonAPage.propTypes = {
     nextButtonLabel: PropTypes.string,
   }).isRequired,
   fields: PropTypes.exact({
-    personARequestsFahrkostenpauschale: fieldPropType,
+    personARequestsFahrkostenpauschale: selectionFieldPropType,
```

Comment:
Why did you have to change this? Shouldn't we then also have to change the fieldType in `FahrkostenpauschalePage.js`?